jas/commons_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	jfrogappsconfig "github.com/jfrog/jfrog-apps-config/go"
```

Comment:
What is the difference between common_test.go and commons_test.go?
Can we join them together to one file?

Reply:
yep, a different PR is coming soon :)